.NET/Microsoft.Recognizers.Text.DateTime/Turkish/Parsers/TurkishDateTimePeriodParserConfiguration.cs

```diff
@@ -24,7 +24,8 @@ public class TurkishDateTimePeriodParserConfiguration : BaseDateTimeOptionsConfi
         public TurkishDateTimePeriodParserConfiguration(ICommonDateTimeParserConfiguration config)
             : base(config)
         {
-            TokenBeforeDate = DateTimeDefinitions.TokenBeforeDate;
+            // @TODO replace token with list of tokens (in Turkish many different suffixes can be used)
+            TokenBeforeDate = DateTimeDefinitions.TokenBeforeDate1;
```

Comment:
Will this happen before this PR can merge?

Reply:
yes, updated this and modified BaseDateTimePeriodParser to work with a list of tokens (when before/after flag is true)